organizing_hub/views/call_views.py

```diff
@@ -121,6 +129,46 @@ def can_make_call_for_campaign(user, call_campaign):
     return False
 
 
+def get_or_create_callers(caller_emails):
+    """
+    Get or create callers from a list of caller emails
+
+    Parameters
+    ----------
+    emails : list of str
+        List of email addresses
+
+    Returns
+        -------
+        Callers list
+            Returns list of ints corresponding to callprofile IDs
+    """
+
+    caller_ids = []
+
+    if caller_emails is not None and caller_emails != '':
+        caller_emails = [email.strip() for email in caller_emails.split(",")]
+
+        for email in caller_emails:
+            # TODO: support multiple accounts per email
+            (user, created) = User.objects.get_or_create(
+                email=email,
+                defaults={
+                    'username':email
+                }
+            )
+
+            if not hasattr(user,'bsdprofile'):
```

Comment:
I still need to move the BSD Profile creation out, or this entire function out. Thoughts on where to put this or how to structure? Perhaps a function that handles this in organizing hub that's imported here?

Reply:
Why do you need to move it out? Previously I asked to move it because it was in the calls app form layer. Now it's in the organizing hub app views layer, which is much better.

Reply:
Wait I had forgotten I moved it here, disregard actually.

Reply:
Although in general I think we should explore some sort of service layer which is not models and not views. I have been putting service api methods in models or views, but really would be nice to have some sort of service api layer in these apps. Not sure what is best practice in django.

Reply:
Maybe move to post-save signal?